eduvpn/ui/ui.py

```diff
@@ -493,6 +503,12 @@ def enter_ConfiguringConnection(self, old_state, new_state):
     def exit_ConfiguringConnection(self, old_state, new_state):
         self.hide_loading_page()
 
+    @transition_edge_callback(ENTER, network_state.ConnectedState)
+    def enter_ConnectedState(self, old_state, new_state):
+        is_expanded = self.connection_info_expander.get_expanded()
```

Comment:
Maybe the function should be named `is_expanded` if you feel the need to use a named variable here.

Reply:
I cannot change this, this is the gtk api

Reply:
Yes, sorry

Reply:
No worries!